package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "leveldown",
   "description": "A Node.js LevelDB binding, primary backend for LevelUP",
-  "version": "0.10.2",
+  "version": "0.10.3",
```

Comment:
should leave this to the releaser (me) btw, it's part of the `npm version` workflow and the commit gets signed as well.